src-docs/src/views/datagrid/datagrid_styling_example.js

```diff
@@ -183,7 +183,9 @@ export const DataGridStylingExample = {
       text: (
         <p>
           When wrapped inside a container, like a dashboard panel, the grid will
-          start hiding controls and adopt a more strict flex layout
+          start hiding controls and adopt a more strict flex layout.
+          Use the `minSizeForControls` prop to control the min width to
```

Comment:
In our docs, we use **EuiCode** to wrap values normally distinguished with backticks.

```suggestion
          Use the <EuiCode>minSizeForControls</EuiCode> prop to control the min width to
```

This change might require a new line break to make prettier happy